Updated mkdocs base image

benarent opened the pull request:
This PR updates our base image for mkdocs. The big upgrades are

- Bump 1.x for MKDocs https://www.mkdocs.org/about/release-notes/#version-10-2018-08-03 
- Bump [Markdown](https://python-markdown.github.io/) from 2x to 3x https://python-markdown.github.io/change_log/release-3.1/ 
- Bump [Click](https://pypi.org/project/click/) to 7.0 
- Add https://pypi.org/project/mkdocs-markdownextradata-plugin/ 

I removed some old draw.io images and combined the README with the intro.

benarent committed:
WIP to update MkDocs base theme.

benarent committed:
Fix 4.0 and 4.2

benarent committed:
Bump copyright

benarent committed:
Removing these out of data diagrams

benarent committed:
Combine Readme with Intro

benarent committed:
Remove old quickstart page

benarent committed:
Delete aws_ent_guide.md

benarent committed:
Remove Readme combined with intro.

benarent committed:
Use official quay image

benarent committed:
Fix for #2813

webvictim commented on docs/4.2/enterprise/ssh_fips.md:
```diff
@@ -136,6 +138,12 @@ cryptographic module (BoringCrypto) and fails to start if it was not.
 
+### FedRAMP Audit Log
+
+. At the close of a connection (close of a *srv.ServerContext) the total data transmitted and received is emitted to the Audit Log.
```
Full stop at the start here? Should probably be `*`

webvictim commented on docs/4.2/enterprise/ssh_fips.md:
```diff
@@ -136,6 +138,12 @@ cryptographic module (BoringCrypto) and fails to start if it was not.
+. At the close of a connection (close of a *srv.ServerContext) the total data transmitted and received is emitted to the Audit Log.
+
+
+
```
Extra space

webvictim commented on docs/4.1.yaml:
```diff
@@ -9,20 +9,22 @@ copyright: Gravitational Inc, 2016-19
   - admonition
   - def_list
   - footnotes
+  #- codehilite
```
Commented out?

benarent replied:
ow, yeah. Removed for now. Will open another PR with updates to our code highlighting.

webvictim commented on docs/4.1/quickstart.md:
````diff
@@ -100,15 +100,15 @@ We've got Teleport running but there are no users recognized by Teleport Auth
     this tutorial!
 
-``` bash
+```
````
This is no longer bash?

benarent replied:
Strange, I've fixed that.

webvictim reviewed (commented):
Lots of red in this diff - all dead code?

benarent committed:
Update quickstart

benarent committed:
Can to index vs intro as first page.

benarent committed:
Remove extra deleted / spaces

benarent committed:
Merge branch 'master' into benarent/feature/bump-mkdocs

webvictim reviewed (approved):
LGTM assuming `milv` has nothing to report :)

benarent committed:
Merge branch 'master' into benarent/feature/bump-mkdocs

benarent committed:
HACK <- Tmp fix need to work on better fix for this.

benarent committed:
URL  changed in MkDocs

benarent committed:
Remove old Index.html page and it's strange redirect.

benarent commented:
Extra Notes:
When testing on staging I found that sidebar links didn't work, I've fixed this. The Theme switcher had the same redirect issue . I've hacked this fix for now https://github.com/gravitational/teleport/pull/3314/commits/6f3667454f900e0e0ea912dab29b09e09543fdaa but we'll want to fix this. 

I've also removed the strange '/index.html' page redirect thing that me and @russjones looked it. One less redirect 🙌 .

benarent committed:
Removed extra site-url

benarent committed:
Merge branch 'master' into benarent/feature/bump-mkdocs

benarent committed:
Fixed Search and Version Switcher

benarent committed:
Merge branch 'benarent/feature/bump-mkdocs' of https://github.com/gravitational/teleport into benarent/feature/bump-mkdocs

benarent committed:
Optional highlight.js

benarent committed:
Fix App Switcher

benarent committed:
Merge branch 'master' into benarent/feature/bump-mkdocs
